### ISSUE TITLE
Feature Request: Use np.sqrt() instead of math.sqrt()

### ISSUE THREAD

**Issue by Idontknowwhatimdoing1234** [CalebBell/fluids#41]
It would be nice if  the usage of math.sqrt would be replaced by the use of np.sqrt().

If one is to evaluate Arrays of Data the math.sqrt() Function will raise:
TypeError: cannot convert the series to <class 'float'> 
even though a series with the type of float is given.

It would make using the fluid module more convenient.
![image](https://user-images.githubusercontent.com/63591722/134827682-d0fb232b-9af3-4749-b076-2d22289a4525.png)
![image](https://user-images.githubusercontent.com/63591722/134827696-ef0c82b2-cff3-4df5-9b82-6a34d1923041.png)

**Comment by Idontknowwhatimdoing1234** [comment]
Solution:  (Sorry i forgot to append)
![image](https://user-images.githubusercontent.com/63591722/134827708-4027d088-1b9d-46c7-91ac-d03493210aa6.png)

**Comment by Idontknowwhatimdoing1234** [comment]
Sorry again i forgot to mention i discoverd the error while trying to calculate the Froude number!

**Comment by CalebBell** [comment]
Hi Idontknowwhatimdoing1234,

Thank you for reaching out and your interest in fluids. Support for numpy arrays is provided in fluids according to the following documentation: https://fluids.readthedocs.io/fluids.vectorized.html
If you are on a platform that supports numba as well, you can accelerate the fluids library with it as shown here: https://fluids.readthedocs.io/fluids.numba.html#numpy-support-fluids-numba-vectorized
You can see another discussion of this issue here: https://github.com/CalebBell/fluids/issues/13 and another about obtaining maximum speed here: https://github.com/CalebBell/chemicals/issues/27 
The TL;DR is that I am happy with these options instead of using numpy, and while numpy can be quite convenient, for scalar calls it slows things down unacceptably so I chose not to use it. 
I wish that weren't the case, but it is the case today.

Sincerely,
Caleb

**Comment by Idontknowwhatimdoing1234** [comment]
Hello Caleb,

thank zou for zour response. That one is completly on me i didnt see that there was the possibility to vectorize it! Userneame checks out i would say :D

Thank zou and keep up the great work!
